tests/fiaas_deploy_daemon/deployer/kubernetes/test_deployment_deploy.py

```diff
@@ -452,17 +453,27 @@ def create_expected_deployment(config,
     return deployment
 
 
-def create_environment_variables(infrastructure, global_env=None, version="version", environment=None):
+def create_environment_variables(config, global_env=None, version="version", environment=None):
     _env_variables = {
-        'ARTIFACT_NAME': 'testapp',
         'LOG_STDOUT': 'true',
-        'VERSION': version,
-        'FIAAS_INFRASTRUCTURE': infrastructure,
+        'FIAAS_INFRASTRUCTURE': config.infrastructure,
         'LOG_FORMAT': 'json',
-        'IMAGE': 'finntech/testimage:' + version,
         'CONSTRETTO_TAGS': _create_constretto_tag(environment),
     }
 
+    _managed_env_variables = {
```

Comment:
I am not in love with this being such a repetition of the logic being tested 😞 
Maybe some tests of the _make_env directly could help keep the ordering/priority clear?

Or maybe a separate test that simply shows that setting 'VERSION' in the app's env will work when the flag is set (i.e. will end up being different to FIAAS_VERSION) would be good.

Reply:
yep, I agree on better not rely on the same logic in the tests. You mean something like https://github.com/fiaas/fiaas-deploy-daemon/pull/108/commits/54b3842a2c86ba31f34aa25241f696b03ca0f420?

Reply:
ping @gregjones 🙏